www/src/components/profile/Profile.tsx

```diff
@@ -44,6 +45,10 @@ export function Profile() {
   const [avatarFile, setAvatarFile] = useState<any>()
   const [mutation, { loading }] = useMutation(UPDATE_USER, {
     variables: { attributes: { name, email, avatar: avatarFile } },
+    onCompleted: ({ updateUser: { id, email } }) => {
```

Comment:
I'm pretty sure this is unnecessary, it'll be plumbed into the use context after update

Reply:
Yeah that is still from before I added it in the `src/components/Plural.tsx`.